### PR TITLE
doc(skel) make the skeleton Chart generic

### PR DESCRIPTION
**Pull request by slack**
Make the skeleton more generic. Prompt for the things we would like to see.

Might also be a good opportunity to expand the skeleton to include a Service definition, so we can explain our approach to labels in situ.

### CODE REVIEW

File: skel/manifests/example-pod.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 kind: Pod
 metadata:
-  name: alpine
+  name: example-pod
 spec:
   restartPolicy: Never
   containers:
-  - name: alpine
+  - name: example
   image: "alpine:3.2"
```
Comment (slack) [inline skel/manifests/example-pod.yaml]: One question is should the skeleton chart actually "work"? Debated about setting this to `scratch` and a random command.

Comment (technosophos) [inline skel/manifests/example-pod.yaml]: I don't know if that's a terribly big deal, but it certainly wouldn't hurt. That said... could you _do_ "scratch" and a random command? I don't think scratch has any commands? Maybe "alpine:3.2" with something like "sleep 900" or something?

Comment (slack) [inline skel/manifests/example-pod.yaml]: `scratch` + command wouldn't work. To help prompt folks writing a chart, we may  as well leave this as something that does something, even if it is sleep.

**Comment by slack** [comment]
Rebased again.

**Comment by slack** [comment]
Rebased.

**Pull request merged**